react/features/mobile/navigation/middleware.js

```diff
@@ -75,18 +26,12 @@ function _setRoom({ dispatch, getState }, next, action) {
  */
 function _conferenceFailed({ dispatch, getState }, next, action) {
     const state = getState();
-    const isWelcomePageEnabled = isWelcomePageAppEnabled(state);
     const { error } = action;
 
     // We need to cover the case where knocking participant
     // is rejected from entering the conference
     if (error.name === JitsiConferenceErrors.CONFERENCE_ACCESS_DENIED) {
-        if (isWelcomePageEnabled) {
-            navigateRoot(screen.root);
-        } else {
-            // For JitsiSDK, WelcomePage is not available
-            _sendReadyToClose(dispatch);
-        }
+        goBackToRoot(state, dispatch);
```

Comment:
Ditto, please test this but I think we should `appNavigate`. Test having video on. If you end up with the green camera icon on the welcome page the tracks weren't properly destroyed.

Reply:
Tested with camera on. Looks good.